pkg/util/version/const.go

```diff
@@ -38,6 +38,10 @@ var DefaultInstallStreams = map[int]*Stream{
 		Version:  NewVersion(4, 11, 44),
 		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:52cbfbbeb9cc03b49c2788ac7333e63d3dae14673e01a9d8e59270f3a8390ed3",
 	},
+	12: {
+		Version:  NewVersion(4, 12, 17),
+		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:7ca5f8aa44bbc537c5a985a523d87365eab3f6e72abc50b7be4caae741e093f4",
+	},
```

Comment:
I know it's not meant to be merged, but can we set the min version in 4.12 we're installing to 4.12.23+?  The FIPS fixes are in that version and beyond so we shouldn't offer any lower version as an install target.

Reply:
Changed to 4.12.25